ethtxmanager/ethtxmanager.go

```diff
@@ -119,6 +119,22 @@ func (c *Client) Add(ctx context.Context, owner, id string, from common.Address,
 	return nil
 }
 
+func (c *Client) AddCommitment(ctx context.Context, batchNum uint64, celestiaHeight uint64, commitment []byte, dbTx pgx.Tx) error {
```

Comment:
I don't understand the purpose of this function. The ethTxManager is used to send txs to L1:

- aggregator / sequencesender call the Add method to request the sending of a tx
- ethTxManager service will take the request and work until the tx is confirmed on L1